streaming/src/java/org/apache/hive/streaming/HiveStreamingConnection.java

```diff
@@ -217,6 +216,11 @@ public static Builder newBuilder() {
     private int transactionBatchSize = DEFAULT_TRANSACTION_BATCH_SIZE;
     private boolean streamingOptimizations = DEFAULT_STREAMING_OPTIMIZATIONS_ENABLED;
     private RecordWriter recordWriter;
+    private Long writeId;
+    private Integer statementId = -1;
+    private boolean manageTransactions = true;
+    private Table tableObject;
+    private Boolean isPartitioned;
```

Comment:
nit: should this be object? I don't see null or not null checks anywhere for this.